src/components/behandlerdialog/meldinger/MeldingerISamtale.tsx

```diff
@@ -54,16 +68,28 @@ const MeldingTilBehandler = ({ melding }: MeldingInnholdProps) => {
 
 interface MeldingerISamtaleProps {
   meldinger: Melding[];
+  skalHenteVedlegg: boolean;
 }
 
-export const MeldingerISamtale = ({ meldinger }: MeldingerISamtaleProps) => {
+export const MeldingerISamtale = ({
+  meldinger,
+  skalHenteVedlegg,
+}: MeldingerISamtaleProps) => {
   return (
     <StyledWrapper>
       {meldinger.map((melding: Melding, index: number) => {
         return melding.innkommende ? (
-          <MeldingFraBehandler melding={melding} key={index} />
+          <MeldingFraBehandler
+            melding={melding}
+            skalHenteVedlegg={skalHenteVedlegg}
+            key={index}
+          />
         ) : (
-          <MeldingTilBehandler melding={melding} key={index} />
+          <MeldingTilBehandler
+            melding={melding}
+            skalHenteVedlegg={skalHenteVedlegg}
```

Comment:
Trengs denne propen her? Vi skal vel aldri hente noen vedlegg for melding til behandler.

Reply:
Nei, egentlig ikke 🤔 
MeldingTilBehandler og MeldingFraBehandler komponentene har samme type tho, men kan jobbe seg rundt det 👍🏼